Release/src/http/listener/http_server_asio.cpp

```diff
@@ -520,7 +520,9 @@ void hostport_listener::start()
     auto& service = crossplat::threadpool::shared_instance().service();
     tcp::resolver resolver(service);
     // #446: boost resolver does not recognize "+" as a host wildchar
-    tcp::resolver::query query = ("+" == m_host) ? tcp::resolver::query(m_port) : tcp::resolver::query(m_host, m_port);
+    tcp::resolver::query query = ("+" == m_host) ?
```

Comment:
I was going to make this same PR!

The following would be shorter if it compiled cleanly everywhere?

```diff
-    tcp::resolver::query query = ("+" == m_host) ? tcp::resolver::query(m_port) : tcp::resolver::query(m_host, m_port);
+    tcp::resolver::query query = ("+" == m_host) ? tcp::resolver::query(m_port, {}) : tcp::resolver::query(m_host, m_port, {});
```